addons/mail/static/src/model/model_manager.js

```diff
@@ -269,6 +269,7 @@ class ModelManager {
                 Model.fields[fieldName] = field;
             } else {
                 Object.assign(Model.fields[fieldName].dependencies, field.dependencies);
+                Model.fields[fieldName].default = field.default;
```

Comment:
Ok for the idea, but in this case the original `default` would be overwritten if `field.default` is `undefined` which is an issue (it means every field patch will clear existing default which is not what we want).

Also note (in regard to your commit tile) that this change is not related to OWL in any way, it's just related to how we manage the data in this module.

Reply:
Ah yes you're right, so is it OK if I add a condition for the undefined case? And if there is some way to do this extension without any changes to Odoo code, I'm also interested in hearing that. We just need this modification implemented somehow.

Reply:
In your case the `default` is not very useful anyway because it is on a field that is also computed, so you could just override `_computeIsSelected` with `registerInstancePatchModel` and return what you need.

Reply:
If the user still needs to be able to change the value without the compute resetting it back though, you'll probably need to do it differently.

I guess in that case overriding `_created` to update the value to false once after creation would be preferable.

Reply:
We implemented the modification with `_created`, thanks alot for the tip!

You can close this if you think using `_created` is the preffered way.

Reply:
Ok, well as it is safer to avoid changing code in stable unless absolutely necessary I will close this PR.

But in the future I'll keep in mind that it would be nice to be able to override the declaration of all field options and not just dependencies.